internal/commands/permission.go

```diff
@@ -106,16 +99,6 @@ func RegisterPermissionCmd(rootCmd *cobra.Command) *cobra.Command {
 		RunE:              lookupResourcesCmdFunc,
 	}
 
-	lookupCmd := &cobra.Command{
-		Use:               "lookup <type> <permission> <subject:id>",
```

Comment:
No longer using a separate command for `lookup`. This is covered by `lookup-resources`.